### ISSUE TITLE
nvda not correctly read the words typed in the power shel

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#8878]
### Steps to reproduce:
I've opted out of speaking characters typed in nvda, the option to speak typed words is enabled.
Dynamic content ads are also enabled.
Open windows power shel.
Enter a word in the test case
Press Space
nvda should announce the word typed in the test case.
What happens is that the nvda read only the first letter t.
now type another word
I tested again.
Now with every typed character, nvda announces space, and by pressing space nvda announces space again.
The expected silence would be when typing characters and pressing space listen to the word you typed.
### Actual behavior:
When you type the first word, nvda announces the first character of the word and from the next words typed in the same line, nvda announces space for each character and announces space at the end of the word.
### Expected behavior:
for any word nvda should be muted when typing the characters and should read the whole typed word by pressing space after it.
#### NVDA Installed/portable/running from source:
install
#### NVDA version:
alfa 16164
#### Windows version:
10 17763.55
#### Name and version of other software in use when reproducing the issue:
power shel
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no

**Comment by PratikP1** [comment]
I can verify this. Along with not reading the word after it’s typed, NVDA announces the first letter of the word currently being typed or the first letter of the word that was just typed. NVDA also does not announce if text is selected when typing.

**Comment by Adriani90** [comment]
@fernando-jose-silva Please test with last NVDA alpha version. Can you still reproduce this issue? I cannot reproduce it anymore.

**Comment by fernando-jose-silva** [comment]
I don't have this problem anymore, thank you very much.

**Comment by Adriani90** [comment]
Thanks for the update, closing as works for me.
